content/en/post/2023-07-10-cross-sign-expiration.md

```diff
@@ -0,0 +1,39 @@
+---
+author: Aaron Gable
+date: 2023-07-10T00:00:00Z
+slug: cross-sign-expiration.md
```

Comment:
```suggestion
slug: cross-sign-expiration
```

The slug in the URL should probably exclude `.md` to keep in line with the rest of our blog URLs